libparsec/crates/crypto/src/sodiumoxide/secret.rs

```diff
@@ -4,9 +4,7 @@ use serde::Deserialize;
 use serde_bytes::Bytes;
 use sodiumoxide::{
     crypto::{
-        pwhash::argon2i13::{
-            derive_key, Salt, MEMLIMIT_INTERACTIVE, OPSLIMIT_INTERACTIVE, SALTBYTES,
-        },
+        pwhash::argon2id13::{derive_key, Salt, MEMLIMIT_MODERATE, OPSLIMIT_MODERATE, SALTBYTES},
```

Comment:
I would suggest to define your own custom values, instead of depending on the one used by `sodiumoxide`.

In fact we already do that for `rustcrypto`, that would simplify synchronizing the value between both implementation.

Reply:
I agree, besides this configuration should be serialized along with the salt and encrypted data (this way we can change this configuration as needed, without breaking compatibility)